stable/fluent-bit/templates/config.yaml

```diff
@@ -89,15 +89,20 @@ data:
         Match *
         Host  {{ .Values.backend.es.host }}
         Port  {{ .Values.backend.es.port }}
-        Logstash_Format On
```

Comment:
This was removed without change. Maybe should be configurable?

Reply:
This was not removed, but transformed into "parameters". See below.